src/analysis/rMTA/calculateEPSILON.m

```diff
@@ -0,0 +1,62 @@
+function epsilon = calculateEPSILON(samples, rxnFBS, varargin)
+% Calculate the minimum required flux change (epsilon) for reactions to be
+% considered as significantly changed with p-value of 0.05
+% 
+% The code below is based on the method presented in:
+%    Yizhak, K., Gabay, O., Cohen, H., & Ruppin, E. (2013). Model-based 
+%    identification of drug targets that revert disrupted metabolism and 
+%    its application to ageing. Nature communications, 4, 2632.
+%
+% USAGE:
+%
+%    epsilon = calculateEPSILON(samples, rxnFBS, 'unique_epsilon', false, 'minimum', 1e-3)
+%
+% INPUT:
+%    samples:           Matrix of sampled fluxes.
+%    rxnFBS:            Array that contains the desired change: Forward,
+%                       Backward and Unchanged (+1;0;-1). This is calculated
+%                       from the rules and differential expression analysis.
+% 
+% OPTIONAL INPUT:
+%    unique_epsilon:    boolean varible. True = unique epsilon, False =
+%                       each reaction has different epsilon (default=false) 
+%    minimum:           Minimun value for epsilon requiered (default=1e-3)
+%
+% OUTPUT:
+%    epsilon            Numeric value or array with the epsilon for the different reactions
+% 
+% .. Authors:
+%       - Luis V. Valcarcel, 06/07/2015, University of Navarra, CIMA & TECNUN School of Engineering.
+% .. Revisions:
+%       - Luis V. Valcarcel, 26/10/2018, University of Navarra, CIMA & TECNUN School of Engineering.
+%       - Francisco J. Planes, 26/10/2018, University of Navarra, TECNUN School of Engineering.
```

Comment:
This function seems to be never called in your tests. 
Would it make sense to write an explicit test Function for it, or modify tthe tests to also run this function?